mk/uvmt/xrun.mk

```diff
@@ -71,7 +71,7 @@ XRUN_RUN_BASE_FLAGS ?= -64bit $(XRUN_GUI) -licqueue +UVM_VERBOSITY=$(XRUN_UVM_VE
 XRUN_GUI         ?=
 XRUN_SINGLE_STEP ?=
 XRUN_ELAB_COV     = -covdut uvmt_$(CV_CORE_LC)_tb -coverage b:e:f:t:u
-XRUN_ELAB_COVFILE = -covfile $(abspath $(MAKE_PATH)/../tools/xrun/covfile.tcl)
```

Comment:
Yes, sort of. It points to the absolute path of the Makefile entrypoint. MAKE_PATH is a variable created in the entry point Makefile's like for instance core-v-verif/cv32e40x/sim/uvmt/Makefile. By searching for files relative to the Makefile location we create a relationship between the Makefile location and the rest of the directory tree.

Changing this to be based on the core-v-verif and core location would be more stable and would allow for Makefiles placed elsewhere to use the targets correctly.

Yes both the old and new way would end up in the $(CORE_V_VERIF)/$(CV_CORE)/sim/tools folder.

Reply:
Cool. Sounds good.